products/postgresql.md

```diff
@@ -22,6 +22,9 @@ identifiers:
 -   repology: postgresql
 -   purl: pkg:generic/postgresql
 -   purl: pkg:docker/library/postgres
+-   purl: pkg:apk/alpine/postgresql
```

Comment:
There is nothing found under https://pkgs.alpinelinux.org/packages?name=postgresql.